### PR TITLE
Feature/dsbd 52

### PR DESCRIPTION
**Pull request by n3op2**
### What

Changed to single binaries execution rather than running in series for cheri and aarch64 within one req

Changes:
- unit test update
- simplified controller as it now executes two commands over ssh rather than 4 (scp + ssh)

- NOTE: this should go after -> https://github.com/digicatapult/morello-api/pull/12 to avoid major conflicts

### CODE REVIEW

File: src/controllers/scenario/index.ts [hunk 1]
```diff
@@ -41,12 +42,9 @@ export class scenario extends Controller implements IScenario {
   }
   
   @Get('{executable}')
-  public async get(@Path() executable: Executables): Promise<ExamplesResult> {
-    this.log.debug(`attempting to execute ${executable} scenario`)
+  public async get(@Path() executable: Executables , @Query() params: string[]): Promise<HostResponse> {
```
Comment (mattdean-digicatapult) [inline src/controllers/scenario/index.ts:45]: does `params` here represent the args passed to the executable?

File: src/controllers/scenario/index.ts [hunk 2]
```diff
@@ -41,12 +42,9 @@ export class scenario extends Controller implements IScenario {
...
+    this.log.debug(`attempting to execute ${executable} scenario with [${params}] arguments`)
     
-    return ({
-      cheri: await this.execute(`${executable}-cheri`),
-      aarch64: await this.execute(`${executable}-aarch64`),
-    })
+    return this.execute(`${executable}-cheri ${params}`)
```
Comment (mattdean-digicatapult) [inline src/controllers/scenario/index.ts]: how does an `aarch64` binary execute?

File: src/controllers/scenario/index.ts (hunk 1 above)
Comment (n3op2) [reply src/controllers/scenario/index.ts:45]: yes! Happy to rename, to be fair was not sure about the name, i think we should call it args. It got stuck in my head  from last week.

File: src/controllers/scenario/index.ts (hunk 2 above)
Comment (n3op2) [reply src/controllers/scenario/index.ts]: 😮‍💨 
M<... this is wrong! let me update. thanks for flagging up

Comment (n3op2) [reply src/controllers/scenario/index.ts]: This has been resolved. It should have been only `${executable}` arch comes from ui

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**